lib/Core/polyfill.js

```diff
@@ -8,4 +8,4 @@ function polyfill(callback) {
   callback();
 }
 
-module.exports = polyfill;
+export default polyfill;
```

Comment:
The TerriaMap PR removes the import of polyfill, can this file be removed completely?